Chat-Ops-Collector/src/main/java/com/capitalone/dashboard/model/ChatOpsRepo.java

```diff
@@ -0,0 +1,56 @@
+package com.capitalone.dashboard.model;
+
+import org.joda.time.DateTime;
+
+/**
+ * CollectorItem extension to store the chatops serverUrl, roomName and authToken.
+ */
+public class ChatOpsRepo extends CollectorItem {
+    private static final String CHATOPS_SERVER_URL = "chatOpsServerUrl";
+    private static final String CHATOPS_ROOM_NAME = "roomName";
+    private static final String CHATOPS_AUTH_TOKEN = "chatOpsAuthToken";
+    private static final String LAST_UPDATE_TIME = "lastUpdate";
+
+    
+    
+    public static String getChatopsAuthToken() {
+		return CHATOPS_AUTH_TOKEN;
+	}
+
+	public static String getChatopsServerUrl() {
+		return CHATOPS_SERVER_URL;
+	}
+
+	public static String getChatopsRoomName() {
+		return CHATOPS_ROOM_NAME;
+	}
+
+	public DateTime getLastUpdateTime() {
+        Object latest = getOptions().get(LAST_UPDATE_TIME);
+        return (DateTime) latest;
+    }
+
+    public void setLastUpdateTime(DateTime date) {
+        getOptions().put(LAST_UPDATE_TIME, date);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+        	return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+        	return false;
+        }
+
+        ChatOpsRepo chatOpsRepo = (ChatOpsRepo) o;
+
+        return getChatopsServerUrl().equals(chatOpsRepo.getChatopsServerUrl()) & getChatopsServerUrl().equals(chatOpsRepo.getChatopsServerUrl()) & getChatopsAuthToken().equals(chatOpsRepo.getChatopsAuthToken());
```

Comment:
use Objects.equals(getChatopsServerUrl(), chatOpsRepo.getChatopsServerUrl()), it handles case of nulls..